src/python/turicreate/toolkits/activity_classifier/_activity_classifier.py

```diff
@@ -259,6 +240,27 @@ def create(dataset, session_id, target, features=None, prediction_window=100,
     else:
         valid_iter = None
 
+    # C++ model
+
+    if USE_CPP:
+
+        name = 'activity_classifier'
+
+        import turicreate as _turicreate
+
+        # Imports tensorflow
+        import turicreate.toolkits.libtctensorflow
+
+        model = _turicreate.extensions.activity_classifier()
+        options = {}
+        options['prediction_window'] = prediction_window
+        options['batch_size'] = batch_size
+        options['max_iterations'] = max_iterations
+        options['verbose'] = verbose
+
+        model.train(dataset, target, session_id, validation_set, options)
+        return ActivityClassifier_beta(model_proxy=model, name=name)
```

Comment:
Why did this move?

Reply:
hmmmm actually I changed the move 
the failing of `test_create_missing_value` can be solved by 
https://github.com/apple/turicreate/blob/d18f03a04eeeb9c3de71a134d1a8682e6c86f766/src/python/turicreate/toolkits/activity_classifier/_activity_classifier.py#L191

and the breaks which relative to auto split train/val data `test_create_auto_validation_set_small`, `test_create_auto_validation_set_string_session_id`, `test_create_auto_validation_set_typical` are solved in 
https://github.com/apple/turicreate/blob/d18f03a04eeeb9c3de71a134d1a8682e6c86f766/src/python/turicreate/toolkits/activity_classifier/_activity_classifier.py#L179